packages/eas-cli/src/devices/actions/create/__tests__/action-test.ts

```diff
@@ -60,6 +61,7 @@ describe(DeviceCreateAction, () => {
         id: 'account_id',
         name: 'foobar',
       };
+      // @ts-expect-error appleTeam is missing properties of AppleTeam GraphQL type
```

Comment:
I wasn't sure how to make this code type check, because the `AppleTeam` type generated from the schema includes many fields not included here (but we don't query all the fields, I guess, so technically the mock is correct.

> Type '{ id: string; appleTeamIdentifier: string; appleTeamName: string; }' is missing the following properties from type 'AppleTeam': account, appleAppIdentifiers, appleDistributionCertificates, applePushKeys, appleProvisioningProfiles ts(2739)

Reply:
Maybe we should change the argument type for `DeviceCreateAction` so it's `Pick<AppleTeam, '...' | ...>` instead of `AppleTeam` if this makes sense?

Reply:
Good idea, I did this.